app/views/pages/home.html.haml

```diff
@@ -83,28 +82,37 @@
             class: 'card-image-fx-normal'
           = image_tag 'slices/resources-red.svg',
             class: 'card-image-fx-hover'
-        %h3.card-heading.card-heading-larger Hack Club's Resources
+        %h3.card-heading.card-heading-taller Our Resources
```

Comment:
Get rid of .card-heading-taller from all 4 card headers now that "Our Resources" can fit on one line.